package.json

```diff
@@ -99,16 +100,18 @@
     "preact-render-to-json": "^3.6.6",
     "prettier": "^1.19.1",
     "redoc-cli": "0.9.6",
-    "webpack-dev-server": "^2.11.3"
+    "webpack-dev-server": "^3.10.3"
   },
   "dependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-transform-react-jsx": "^7.0.0",
     "@github/clipboard-copy-element": "^1.1.2",
-    "@rails/webpacker": "^3.5.5",
-    "babel-plugin-transform-react-jsx": "^6.24.1",
-    "babel-preset-preact": "^1.1.0",
+    "@rails/webpacker": "4.2.2",
+    "babel-preset-preact": "^2.0.0",
     "chart.js": "^2.9.3",
     "clipboard-polyfill": "^2.8.6",
     "codemirror": "^5.52.0",
+    "core-js": "3",
```

Comment:
Babel 6 shipped with core-js 2 out of the box. Now you need to explicitly install core-js and reference it in `babel.config.js`.